app/deps/requirements.txt

```diff
@@ -4,21 +4,23 @@ aws-xray-sdk
 # Intentionally commented boto3. We can just leverage the AWS Lambda Python Runtime cache for boto3 and botocore.
 #boto3
 
-Django==5.2
-djangorestframework==3.15.2
-django-cors-headers==4.6.0
-django-environ==0.11.2
+# App main dependencies
+Django==5.2.3
+djangorestframework==3.16.0
+django-cors-headers==4.7.0
+django-environ==0.12.0
 djangorestframework-camel-case==1.4.2
 drf-spectacular==0.28.0
 
 # See psycopg[binary] or psycopg[c] impl https://www.psycopg.org/psycopg3/docs/basic/install.html
-psycopg[binary]==3.2.3
+psycopg[binary]==3.2.9
 Werkzeug==3.1.3
 libumccr==0.4.1
-cachetools==5.5.0
-serverless-wsgi==3.0.5
+cachetools==6.1.0
+serverless-wsgi==3.1.0
+ulid-py==1.1.0
+
+# FIXME deprecate EventBridge Schema Registry https://github.com/OrcaBus/service-shared-resources/issues/10
 # six and regex required by automatically generated EventBridge code binding
-# for sequencerunstatechange package
 six==1.17.0
 regex==2024.11.6
```

Comment:
TODO remove six, regex when deprecating EB schema registry code binding 

Part of https://github.com/OrcaBus/service-shared-resources/issues/10